### PR TITLE
chore(card): add a11y demonstration and social post like card

### PR DESCRIPTION
**Pull request by bbogdanov**
Signed-off-by: Bogdan Bogdanov <bbogdanov@vmware.com>

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by mathisscott — APPROVED** [review]

File: packages/core/src/card/card.stories.mdx [hunk 1]
```diff
@@ -15,15 +15,11 @@ import '@cds/core/card/register.js';
 ```html
 <cds-card>
```
Comment (Jinnie) [inline packages/core/src/card/card.stories.mdx:16]: Do we need the aria-labelledby here too?

File: .vscode/settings.json [hunk 2]
```diff
@@ -3,5 +3,9 @@
...
     "**/*.element.css.ts": true,
     "**/*.global.css.ts": true
-  }
-}
\ No newline at end of file
+  },
+  "i18n-ally.localesPaths": [
+    "packages/core/src/internal/i18n",
+    "packages/angular/projects/clr-angular/src/utils/i18n"
+  ]
```
Comment (coryrylan) [inline .vscode/settings.json]: Should this be defined in the project repo or per local machine?

File: packages/core/src/card/card.stories.ts [hunk 3]
```diff
@@ -249,39 +239,67 @@ export function WithImage() {
...
-          Card Title
-        </div>
-
-        <cds-divider cds-card-remove-margin></cds-divider>
-
+export function SocialPost() {
+  ClarityIcons.addIcons([
+    'batman',
+    renderIcon(`
+    <svg viewBox="29.25 -183.153 985.166 985.166" xmlns="http://www.w3.org/2000/svg">
```
Comment (coryrylan) [inline packages/core/src/card/card.stories.ts]: Running the svg through https://jakearchibald.github.io/svgomg/  will save up some space

File: .vscode/settings.json (hunk 2 above)
Comment (bbogdanov) [reply .vscode/settings.json]: Tried to removed it but VSCode constantly generated it after I tried delete it. Strange behaviour but didn't dig into what's causing it.

File: packages/core/src/card/card.stories.ts (hunk 3 above)
Comment (bbogdanov) [reply packages/core/src/card/card.stories.ts]: from `9.1k` to `3.7k`. Thanks!

**Review by mathisscott — APPROVED** [review]

**Review by hippee-lee — APPROVED** [review]
nananana -> lgtm.

**Comment by bbogdanov** [comment]
#5563

**Pull request merged**